src/bundler/bundle_v2.zig

```diff
@@ -1092,9 +1087,8 @@ pub const BundleV2 = struct {
                     const source = &sources[index];
                     var pathname = source.path.name;
 
-                    const rel = bun.path.relative(this.bundler.options.root_dir, source.path.text);
-                    if (rel.len > 0 and rel[0] != '.')
-                        pathname = Fs.PathName.init(rel);
+                    // TODO: outbase
+                    pathname = Fs.PathName.init(bun.path.relativePlatform(this.bundler.options.root_dir, source.path.text, .loose, false));
```

Comment:
I don't know exactly where `pathname` ends up, but should this have `.posix` for the platform to make sure there's only forward slashes?